### ISSUE TITLE
Does the food logging still work?

### ISSUE THREAD

**Issue by JPFrancoia** [jakethekoenig/Fitbit-Coffee-Logger#3]
Hi,
Just installed the app on my Sense. I love its simplicity. However, it looks like it's not sending the number of coffees I had today to the app. Does it happen once a day or is it instantaneous?
If it's supposed to happen instantly, do you know if there was an API change that could have broken this feature?

Best

**Comment by jakethekoenig** [comment]
Just checked and it still works for me on my Fitbit versa 3. Are you logged into Fitbit? Unfortunately you can't (at the time I made the app this may, hopefully, have changed) use the food API directly from an app you need to make a web request which needs to be authenticated with oauth. If you open the app's settings which you can find in your Fitbit app under your device you should see a login button. After logging in it should work. Unfortunately the logged in session is only like a day I believe.

**Comment by JPFrancoia** [comment]
> Are you logged into Fitbit?

I think so. I had clicked on the login button and I went through the auth flow. That's what I see now in the app:

![Screenshot_2024-05-31-14-53-14-416_com fitbit FitbitMobile](https://github.com/jakethekoenig/Fitbit-Coffee-Logger/assets/2399090/359493b8-0be8-41ec-be2b-1bbe68ad554b)

When I click on it again,  a web page opens but instantly redirects me to the app.

> After logging in it should work. Unfortunately the logged in session is only like a day I believe.

Does the app backfills? Say for example I record my coffees for a week, every day. On sunday I decide to login. Will the app uploads all the coffee tracking of the week?
